src/components/MenuPanel/MenuPanel.js

```diff
@@ -18,13 +19,25 @@ import MenuPanelAppsLoader from './MenuPanelAppsLoader'
 import NotificationAlert from '../Notifications/NotificationAlert'
 import OrganizationSwitcher from './OrganizationSwitcher/OrganizationSwitcher'
 import AppIcon from '../AppIcon/AppIcon'
+import IconArrow from '../../icons/IconArrow'
 
 const APP_APPS_CENTER = staticApps.get('apps').app
 const APP_HOME = staticApps.get('home').app
 const APP_PERMISSIONS = staticApps.get('permissions').app
 const APP_SETTINGS = staticApps.get('settings').app
 const SHADOW_WIDTH = 15
 
+const systemAppsOpenedState = {
+  key: 'SYSTEM_APPS_OPENED_STATE',
+  isOpen: function() {
+    const val = window.localStorage.getItem(this.key)
+    return typeof val === 'string' ? JSON.parse(val) : false
+  },
+  set: function(val) {
+    window.localStorage.setItem(this.key, val)
+  },
+}
```

Comment:
Maybe we could avoid implicit type conversion here? So that we don’t have to worry about handling JSON parsing errors.

I would also drop `window`:

```jsx
const systemAppsOpenedState = {
  key: 'SYSTEM_APPS_OPENED_STATE',
  isOpen: () => localStorage.getItem(this.key) === '1',
  set: opened => localStorage.setItem(this.key, opened? '1' : '0'),
}
```

Reply:
For the time being I don't see a reason to implement the second suggestion, I did implement the first one.

Reply:
Yes of course 👍